c2corg_api/views/document.py

```diff
@@ -30,3 +31,64 @@ def _create_new_version(self, document):
         DBSession.add(meta_data)
         DBSession.add_all(versions)
         DBSession.flush()
+
+    def _update_version(self, document, comment, update_type, changed_langs):
+        assert update_type != UpdateType.NONE
+
+        meta_data = HistoryMetaData(comment=comment)
+        archive = self._get_document_archive(document, update_type)
+
+        cultures = \
+            self._get_cultures_to_update(document, update_type, changed_langs)
+        locale_versions = []
+        for culture in cultures:
+            locale = document.get_locale(culture)
+            locale_archive = self._get_locale_archive(locale, changed_langs)
+
+            version = DocumentVersion(
+                document_id=document.document_id,
+                culture=locale.culture,
+                version=999,
+                document_archive=archive,
+                document_locales_archive=locale_archive,
+                history_metadata=meta_data
+            )
+            locale_versions.append(version)
+
+        DBSession.add(archive)
+#         DBSession.add_all(archive_locales)
+        DBSession.add(meta_data)
+        DBSession.add_all(locale_versions)
+        DBSession.flush()
```

Comment:
This makes the tests fail, because the changes are not visible. Though I would have expected that an implicit commit is done?!